doc/menu_lua_api.txt

```diff
@@ -229,6 +229,10 @@ core.get_min_supp_proto()
 core.get_max_supp_proto()
 ^ returns the maximum supported network protocol version
 
+Other:
+core.open_url(url)
+^ opens the URL in a web browser. Must begin with http:// or https://
```

Comment:
Also needs a description of what kind of URLs are allowed (direct domain links only).